R/format.R

```diff
@@ -36,6 +36,9 @@ knit_print.css <- function(x, options, ...) {
 }
 
 knit_print.sass <- function(x, options, ...) {
+  if (isTRUE(options$class.output %in% c('scss', 'sass'))) {
+    return(as.character(x))
+  }
```

Comment:
Can you remind me what this is for?

Reply:
Oh this is for your docs, basically?

Reply:
Yea, it's analogous to what we already do for `knit_print.css()`:

https://github.com/rstudio/sass/pull/36/files#diff-2d8e8f1469f39259a828f967d4dc9cefR30-R33